src/cmd/update.go

```diff
@@ -1,46 +1,19 @@
 package cmd
 
 import (
-	"os"
-
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 )
 
-var updateDataFile string
-
 var updateCmd = &cobra.Command{
 	Use:   "update",
-	Short: "Update resources in OpsLevel",
-	Long:  "Update resources in OpsLevel",
+	Short: "Update resources or events from a file or stdin",
+	Long:  "Update resources or events from a file or stdin",
 }
 
 func init() {
 	rootCmd.AddCommand(updateCmd)
 
-	updateCmd.PersistentFlags().StringVarP(&updateDataFile, "file", "f", "-", "File to read update from. If '.' then reads from './data.yaml'. Defaults to reading from stdin.")
-	viper.BindPFlags(createCmd.Flags())
```

Comment:
I think the reference to `createCmd` here is a mistake - we want this flag binded globally if we're using update.